packages/block-editor/src/components/responsive-block-control/README.md

```diff
@@ -27,44 +27,20 @@ import {
 	InspectorControls,
 	__experimentalResponsiveBlockControl as ResponsiveBlockControl,
 } from '@wordpress/block-editor';
-import {
-	DimensionControl,
-} from '@wordpress/components';
```

Comment:
This `ResponsiveBlockControl` component itself is also obsolete, but updating the example in this readme anyway.

Here is a story if you want to see what this component does:

```jsx
/**
 * WordPress dependencies
 */
import { useState } from '@wordpress/element';

/**
 * Internal dependencies
 */
import ResponsiveBlockControl from '../';

export default {
	title: 'BlockEditor/ResponsiveBlockControl',
	component: ResponsiveBlockControl,
};

export const Default = () => {
	const [ isResponsive, setIsResponsive ] = useState( false );

	return (
		<ResponsiveBlockControl
			title="Padding"
			property="padding"
			isResponsive={ isResponsive }
			onIsResponsiveChange={ () => {
				setIsResponsive( ! isResponsive );
			} }
			renderDefaultControl={ ( labelComponent, viewport ) => {
				return (
					<>
						<label htmlFor={ viewport.id }>
							{ labelComponent }
						</label>
						<input type="number" id={ viewport.id } />
					</>
				);
			} }
		/>
	);
};
```